config/redis.js

```diff
@@ -0,0 +1,22 @@
+'use strict';
+
+const config = require('config');
+
+const redisConfig = config.get('redis');
+
+const queuePrefix = redisConfig.prefix || '';
+
+const connectionDetails = {
+    pkg: 'ioredis',
+    host: redisConfig.host,
+    options: {
+        password: redisConfig.password
+    },
+    port: redisConfig.port,
+    database: 0
+};
+
+module.exports = {
```

Comment:
If this module is going to be refactored out, we should also refactor the tests so they're simpler